docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,79 +2,39 @@
+    - Therefore, changes such as updated names of imported entities might be shown in the GH diff. 
+    - The new terms should be available for logical definitions in Protege. Therefore one can also edit the file too, but changes not manually made could be expected (see previous comment)
+
+**Note when adding a new NCBITaxon**
+Check if the new NCBI Taxon identifier(s) exist in [https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt](https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt). 
     - if the identifiers are not in the file, create a PR in this repo to include the new identifiers
-
     Note: this file contains CURIEs (not IRIs) so the identifier should be added in this format `NCBITaxon:1`
 
-    This additional step is needed since we are not using NCBI Taxon directly, but the OBO slim, and this file is the seed of the NCBITaxon slim.
+This additional step is needed since we are not using NCBI Taxon directly, but the OBO slim, and this file is the seed of the NCBITaxon slim.
  
----
-#### Detailed instructions for adding a new gene
-
-1. Find the gene in HGNC that you need to add:
-https://www.genenames.org/. Copy the ID (for example, 8965)
-2. Open the `src/ontology/imports/hgnc_terms.txt` file
-3. Add a new line to the file: http://identifiers.org/hgnc/[your ID], for example http://identifiers.org/hgnc/8965
-5. Run import:
-`sh run.sh make refresh-merged`
-4. Close Protege and open again
-5. Edit in Protege
-6. In your diff, you will see changes to src/ontology/imports/hgnc_terms.txt and src/ontology/imports/merged_import.owl
-7. Commmit changes on a branch and create a pull request.
-
-Note: if an import does not have a `src/ontology/imports/*_terms.txt` file (e.g. for adding NCIT terms). The term(s) to import should be added to src/ontology/imports/manual_seed.txt.  When updating the import, the term(s) will seed the import files, which are pulled in when refreshing modules.
 
-### Alternate instructions
 
-#### Add classes from external ontologies using a Text Editor
+### Alternate approaches
+These alternate approaches are mentioned here for documentation purposes. These should **only be used by experienced curators**, and followed by a clean-up of class declarations in mondo-edit.obo 
 
-1. Close Protege
-1. Checkout master
-1. Create a new branch
-1. Open mondo-edit.obo with a text editor like Sublime or TextEdit
-1. Add axiom in mondo-edit.obo *text file*
-1. **For example:**  relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/129
-1. Save text file
-1. open mondo-edit.obo in Protege
-1. File -> save as (obo format, save as mondo-edit.obo)
-1. Replace existing file  
-1. Check diff
-1. Commit/Push
+- **Add the required axiom, using classes from external ontologies, the in mondo-edit.obo *text file* using a Text Editor**
+ For example:  relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/129
```

Comment:
If an experienced curator follows this method correctly, does making this change in the Text Editor work and not cause any further downstream issues? I'm unclear about Line 31 "and followed by a clean-up of class declarations in mondo-edit.obo", which sounds like both of these alternative approaches are fine, but just need some extra work.

Reply:
I do not know what type of clean-up will be needed. My understanding is that it will be updated after the next import, but I don't know if there is anything else.

Reply:
Ok, my impression was that there wasn't really a way to do the clean-up. @matentzn what is the situation here?

Reply:
In earlier comments, there was a decision to remove the "alternate approaches" section.